### DISCUSSION TITLE
Enhancement for the #clear-data-on-exit flag honour the chrome://settings/clearBrowserData choices

### DISCUSSION THREAD

**Discussion by Kees1958** [Eloston/ungoogled-chromium#2934] [Ideas]
First of all, thanks for adding this flag. 

Secondly: It would be nice when the flag would honour the Delete Browsing data settings (like Edge and Brave have implemented)
chrome://settings/clearBrowserData

**Comment by PF4Public** [comment]
Duplicate of #1923?

**Comment by Kees1958** [comment]
@PF4Public sorry you are right. It is a duplicate (I simple looked for other keywords)
